polylaue/ui/regions_navigator/dialog.py

```diff
@@ -101,14 +103,12 @@ def on_mouse_pressed(
 
         pos = self.image_view.getView().mapSceneToView(event.scenePos())
 
-        initial_size = (1, 1)
+        position: WorldPoint = np.array((pos.x(), pos.y()), dtype=np.float32)
```

Comment:
I personally think we should, in general, just let it use the default `float64` unless, through profiling and testing, we find that `float32` is noticeably faster for a segment of code.

The `float32` can more easily cause an accumulation of floating point errors when doing arithmetic, which would be a much bigger problem than running microseconds slower from using `float64`.

But let's just leave this for now.